sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -2476,7 +2477,7 @@ class SQLQuerySuite extends QueryTest with SharedSQLContext {
     withTempDir { dir =>
       val parquetDir = new File(dir, "parquet").getCanonicalPath
       spark.range(10).withColumn("_col", $"id").write.partitionBy("_col").save(parquetDir)
```

Comment:
Since the data format may not be parquet, maybe the directory name should be more generic, like dataDir.

Reply:
Thank you for review, @bersprockets .